### PR TITLE
Refactor Scalar DB Server code

### PR DESCRIPTION
**Pull request by brfrn169**
This is very trivial things. Please take a look!

### CODE REVIEW

File: server/src/main/java/com/scalar/db/server/ServerConfig.java [hunk 1]
```diff
@@ -1,11 +1,8 @@
-package com.scalar.db.server.config;
+package com.scalar.db.server;
```
Comment (brfrn169) [inline server/src/main/java/com/scalar/db/server/ServerConfig.java:1]: I moved `ServerConfig` from the `com.scalar.db.server.config` package to the `com.scalar.db.server` package.

File: server/src/main/java/com/scalar/db/server/ServerModule.java [hunk 2]
```diff
@@ -1,4 +1,4 @@
-package com.scalar.db.server.service;
+package com.scalar.db.server;
```
Comment (brfrn169) [inline server/src/main/java/com/scalar/db/server/ServerModule.java:1]: I also moved `ServerModule` from the `com.scalar.db.server.service` package to the `com.scalar.db.server` package.

**Review by thongdk8 — APPROVED** [review]
LGTM! Thank you!

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**